federator/src/contracts/IBridgeNft.ts

```diff
@@ -0,0 +1,27 @@
+export class IBridgeNft {
+  nftBridgeContract: any;
+
+  constructor(nftBridgeContract: any) {
+    this.nftBridgeContract = nftBridgeContract;
+  }
+
+  getFederation(): any {
+    return this.nftBridgeContract.methods.getFederation();
+  }
+
+  getPastEvents(eventName: string, options: any): any {
+    return this.nftBridgeContract.getPastEvents(eventName, options);
+  }
+
+  getAddress(): string {
+    return this.nftBridgeContract.options.address;
+  }
+
+  getProcessed(transactionHash: string): any {
+    return this.nftBridgeContract.methods.processed(transactionHash).call();
+  }
+
+  getVersion(): string {
+    return this.nftBridgeContract.methods.version();
```

Comment:
Add call to retrieve the version
```suggestion
    return this.nftBridgeContract.methods.version().call();
```

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/257/commits/960a77254356ad684efe45071b291b518e0266f8